playground/module-resolution/__tests__/no-prebundling/module-respolution-no-prebundling.spec.ts

```diff
@@ -6,6 +6,7 @@ import {
 	viteTestUrl,
 } from '../../../__test-utils__';
 
+// TODO: test build
```

Comment:
This doesn't currently pass when the build output is tested. Perhaps @dario-piotrowicz  can look into this in a subsequent PR.

Reply:
sure! sounds good to me 🙂👍